app/pages/schedule.vue

```diff
@@ -22,3 +122,180 @@ useSeoMeta({
+        border-style var(--hover-transition),
+        border-color var(--hover-transition);
+    &.is-active {
+        background-color: var(--gray-900);
+        color: var(--beige-100);
+        border-color: var(--gray-900);
+    }
+    &:hover,
+    &:focus-visible {
+        background-color: var(--beige-100);
+        color: var(--gray-900);
+        border-color: var(--gray-900);
+        border-style: dashed;
+    }
+}
+.schedule-grid {
+    margin-bottom: 64px;
+}
+.timeslot {
+    @media (--md) {
+        display: grid;
+        grid-template-columns: 80px 1fr;
+    }
+    @media (--lg) {
+        &.regular {
+            + .regular {
+                .timeslot-sessions {
+                    border-top: 0;
+                }
+            }
+            &:has(+ .special) {
+                margin-bottom: 24px;
+                .timeslot-sessions {
+                    border-bottom-left-radius: 8px;
+                    border-bottom-right-radius: 8px;
+                }
+            }
+        }
+    }
+    &.special {
+        margin-bottom: 24px;
+        .timeslot-sessions {
+            display: block;
+        }
+        + .regular {
+            .timeslot-sessions {
+                border-top-left-radius: 8px;
+                border-top-right-radius: 8px;
+            }
+        }
+    }
+    &:first-child {
+        .timeslot-sessions {
+            border-top-left-radius: 8px;
+            border-top-right-radius: 8px;
+        }
+    }
+    &:last-child {
+        .timeslot-sessions {
+            border-bottom-left-radius: 8px;
+            border-bottom-right-radius: 8px;
+        }
+    }
+}
+.time {
+    display: block;
+    font-weight: 600;
+    @media (--md-down) {
+        margin-bottom: 16px;
+    }
+    @media (--md) {
+        margin-top: 68px;
+    }
+    @media (--lg) {
+        margin-top: 0;
+        &.has-place {
+            margin-top: 68px;
+        }
+    }
+}
+.timeslot-sessions {
+    border: 1px solid var(--gray-900);
+    margin-bottom: 24px;
+    overflow: hidden;
+    @media (--lg-down) {
+        border-radius: 8px;
+    }
+    @media (--lg) {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        margin-bottom: 0;
+    }
+    .special & {
+        border-radius: 8px;
+    }
+}
+.place {
+    padding: 24px;
+    font-weight: 600;
+    border-bottom: 1px solid var(--gray-900);
+}
+.session {
+    display: flex;
+    flex-direction: column;
+    border-bottom: 1px solid var(--gray-900);
+    @media (--lg) {
+        border-right: 1px solid var(--gray-900);
+        border-bottom: 0;
+    }
+    &:last-child {
+        border-right: 0;
+        border-bottom: 0;
+    }
+}
+.session-cell {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-grow: 1;
+    @media (--lg) {
+        justify-content: center;
+    }
+}
+.to-be-anounced {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--gray-300);
```

Comment:
Augmenter le contrast ici pour l'accessibilité:
```suggestion
    color: var(--gray-600);
```